war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2275,11 +2286,12 @@ if (typeof Form === "object") {
   };
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
according to the docs it should be marked as exported rather than disabling the rule, it would end up with the same result though so non-blocking
```suggestion
/* exported toQueryString */
```

Reply:
This suggestion causes the build to go from passing to failing with

```
  2295:10  error  'toQueryString' is defined but never used  no-unused-vars
```

thereby making it a non-suggestion.

Reply:
right it's because some of the files are modules. Which that feature doesn't work if modules are enabled.

to make it work there would need to be another eslintrc file for the non modular code I think which is likely not worth it.